docker/rnaseq-cgl-pipeline.cwl

```diff
@@ -53,6 +53,127 @@ hints:
     description: "The process requires at least 16G of RAM and we recommend 500GB or storage."
 
 inputs:
+  auto-scale:
```

Comment:
I don't know enough about the CWL spec to provide a review. Not sure if it's worth doing, but adding a test or two that just checks whether the CWL format is correct or not might be worth adding.

Reply:
Good idea; in Jenkins? We moved to a newer cwl parser so that's what triggered it